src/pageObject.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return 'Click on the ' + this.header.logo;
+  }
 };
 
 class ArticlePage extends PageObject {
-
+  get commentButton() {
+    return '[Publish comment] button';
+  }
+  constructor(url, slug) {
+    super(url);
+    this.slug = slug;
+    this.url = `${url}+${slug}`;
```

Comment:
Instead of using '+' for concatenating strings, use template literals to combine strings. For example: `${url}/${slug}` 🚀